src/common/options.cc

```diff
@@ -7400,6 +7400,20 @@ std::vector<Option> get_mds_options() {
     .set_description("target maximum memory usage of MDS cache")
     .set_long_description("This sets a target maximum memory usage of the MDS cache and is the primary tunable to limit the MDS memory usage. The MDS will try to stay under a reservation of this limit (by default 95%; 1 - mds_cache_reservation) by trimming unused metadata in its cache and recalling cached items in the client caches. It is possible for the MDS to exceed this limit due to slow recall from clients. The mds_health_cache_threshold (150%) sets a cache full threshold for when the MDS signals a cluster health warning."),
 
+    Option("mds_cache_autotune", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
+    .set_description("Enable autotuning of mds_cache_memory_limit")
```

Comment:
Generally, the style for short description is `<noun> [modifiers]`, without initial capitalization. How about `autotuner for MDS cache size to stay within mds_memory_target`.

Reply:
Sounds good. Will change it.